lib/clientBuilder.js

```diff
@@ -81,10 +81,14 @@ function OktaAuthBuilder(args) {
   };
 
   sdk.idToken = {
-    authorize: util.bind(token.getToken, sdk, sdk), // deprecated for sessionToken and idp flows
-    verify: util.bind(token.verifyIdToken, sdk, sdk),
-    refresh: util.bind(token.refreshIdToken, sdk, sdk),
-    decode: util.bind(token.decodeToken, sdk) // deprecated
+    authorize: util.deprecateWrap('Use token.getWithoutPrompt, token.getWithPopup, or token.getWithRedirect ' +
+      'instead of idToken.authorize.', util.bind(token.getToken, sdk, sdk), sdk),
+    verify: util.deprecateWrap('Use token.verify instead of idToken.verify',
+      util.bind(token.verifyIdToken, sdk, sdk), sdk),
```

Comment:
Will change based on other comment. I guess I can change `this` for all these bound functions while I'm at it.